passgithelper.py

```diff
@@ -357,13 +357,66 @@ def define_pass_target(
     return pass_target
 
 
-def compute_pass_environment(section: configparser.SectionProxy) -> Mapping[str, str]:
+def compute_pass_environment(
+    section: configparser.SectionProxy,
+) -> Tuple[Mapping[str, str], Path]:
+    """Return dictionary used to setup environment for ``pass`` sub-processes.
+
+    The main task of this function is to determine the password store directory
+    to be used by ``pass``. It does this by:
+
+    1. using the value of ``password_store_dir`` in ``section`` (if defined and
+       non-empty),
+    2. using the value of the ``PASSWORD_STORE_DIR`` environment variable (if
+       defined and non-empty),
+    3. falling back to the default: ``~/password-store``.
+
+    In the next step, a leading ``~`` (tilde) in the resulting path gets
+    replaced by the users HOME directory (using ``Path.expanduser()``).
```

Comment:
Does this include `%USERPROFILE%` etc. on windows. If so, the text should be more generic.

Reply:
> Does this include %USERPROFILE% etc. on windows.

Yes, it does. I didn't want to write an overly long explanation, which is why I added to Path.expanduser hint. I will think about it.

Reply:
Maybe this can be rephrased as

> In the next step, markers for the home directory such as ``~`` or ``%USERPROFILE`` are expanded (using ...

Reply:
> Maybe this can be rephrased as
> 
> > In the next step, markers for the home directory such as `~` or `%USERPROFILE` are expanded (using ...

This is actually not what is happening. Instead, `~` gets expanded to `$HOME` on Unix and to `%USERPROFILE%` on Window. I changed the wording in the doc comment accordingly.